scopes/html/html-docs-app.ts

```diff
@@ -0,0 +1,26 @@
+import { htmlToReact } from './html-to-react';
+import mapObject from 'map-obj'
+// maybe extract React/docs to its own component?
+// @ts-ignore TODO - fix
+import DocsRoot from '@teambit/react/dist/docs/index.js';
+
+export default function htmlDocsRoot (...args: ReactDocsRootParams) {
+	// const reactDocsTemplate = this.options.reactDocsTemplate;
+	const [Provider, componentId, docs, compositions, ...rest] = args;
+	
+	const reactCompositions = mapObject(compositions, (key, value) => [htmlToReact(value), key]);
+
+	return DocsRoot(Provider, componentId, docs, reactCompositions, ...rest);
+  }
+  
+//   TODO - could use Parameters<DocsRoot>
+  type ReactDocsRootParams = [
```

Comment:
this type seems empty.
anyway, its place is in the react aspect not here.
export it from react, and use it above

Reply:
same, but from `@teambit/react.ui.docs-app`